### ISSUE TITLE
macOS Shortcuts

### ISSUE THREAD

**Issue by Sirage48** [supermerill/SuperSlicer#1263]
Replacing my comments on #1247.  As requested:
1. File shortcuts all seem to work and layout the same 56.1 56.2 56.3 (version number still 56.2)

2. Edit shortcuts also still work BUT they have disappeared in 56.3
56.2
![E23562](https://user-images.githubusercontent.com/35128629/120907848-4cc95380-c6a8-11eb-86df-5c0c2e89f2fb.jpg)

56.3
![E23563](https://user-images.githubusercontent.com/35128629/120907856-5eaaf680-c6a8-11eb-8ac6-69f8a9cb68de.jpg)

3. The TAB shortcuts not working except Filament.

4. Under View "Collapse sidebar"  Shift-TAB is unstable and now flickers collapsed - not collapsed and will finish on one or the other depending how many milliseconds you hold the keys.

**Comment by supermerill** [comment]
Yes, i wondered if I should keep "CTRL" or change them with "CMD"
Can i have a list of working shortcut and a list of broken one?

**Comment by supermerill** [comment]
@liffbag
can someone report on what's showing in the menus with  version https://github.com/supermerill/SuperSlicer/actions/runs/922009616 ?
I should have
```
### File
new project :		Ctrl+N
Open project:		Ctrl+O
save project:		Ctrl+S
save project as...:	Ctrl+Alt+S
Import->stl...:		Ctrl+I
Import->config:		Ctrl+L
Import->config from project:	Ctrl+Alt+L
export->ExportGcode:Ctrl+G
export->Send Gcode .. :	Ctrl+Shift+G
export->gcode to sd:Ctrl+U
export-> config:	Ctrl+E
### Edit
Select all:			Ctrl+A
Deselect all:		Esc
Delete selected:	Del
Delete all:			Ctrl+Del
undo:				Ctrl+Z
Redo:				Ctrl+Y
Copy:				Ctrl+C
paste:				Ctrl+V
reload from disk:	F5
Search:				Ctrl+F
###Windows
tabs: Ctrl+1 to Ctrl+6
Print host upqueue:	Ctrl+J
Open new instance:	Ctrl+shift+I
###View
change viewpoint: 0 to 6
show labels:		Ctrl+E
colapse sidebar:	Shift+Tab
###Configuration
preference			Ctrl+P
```

With a screenshot of the edit menu plz.

**Comment by macdarren** [comment]
![Screen Shot 2021-06-09 at 10 09 36 AM](https://user-images.githubusercontent.com/25966982/121399522-5b637500-c90b-11eb-9e32-834c884a8bf9.jpg)

**Comment by macdarren** [comment]
![Screen Shot 2021-06-09 at 10 08 32 AM](https://user-images.githubusercontent.com/25966982/121399744-95cd1200-c90b-11eb-8d68-8ceeca85a99f.jpg)

![Screen Shot 2021-06-09 at 10 08 56 AM](https://user-images.githubusercontent.com/25966982/121399765-9cf42000-c90b-11eb-811e-da469c727c97.jpg)

![Screen Shot 2021-06-09 at 10 09 04 AM](https://user-images.githubusercontent.com/25966982/121399797-a8dfe200-c90b-11eb-83a2-21c2079e9bad.jpg)

![Screen Shot 2021-06-09 at 10 09 12 AM](https://user-images.githubusercontent.com/25966982/121399844-b4cba400-c90b-11eb-8bb6-711e32d6f4dc.jpg)

**Comment by supermerill** [comment]
can someone test if they are working in this build?
https://github.com/supermerill/SuperSlicer/suites/2955019072/artifacts/66525953
C/C++ Nigthly macos #323:

**Comment by Sirage48** [comment]
Preliminary checks indicate nearly every shortcut is now working - thanks.
macOS Big Sur 11.3.1

The only one not working so far is E (View / Show Labels) which is flakey. Either very slow, many seconds, or not responsive. Not a priority to fix I guess.

⌘ 1 - 6 all good
Shift - Tab working which might fix #1266 ??? which was on 11.4 that I think is a development build of Big Sur.

I noticed preferences are ⌘,  vs ⌘P in Prusaslicer which is shown when pressing   "?"
Just an observation - not a request to change.

I will continue.

**Comment by Sirage48** [comment]
I have been through all the 6 Shortcut help screens on macOS and they all seem to work except some of the Gizmos, which I don't usually use so I don't care too much about these.  In the interest of completion or perfection (??):

- ⌘ for one direction did not work

- ⌥ in conjunction with S  scale or R rotate seems redundant as the default operation uses the centre anyway
       A mystery,  someone else might help (or educate me).

In summary so far all the obvious shortcuts now OK on my Mac with nightly 323  = 2.3.56.4

**Comment by supermerill** [comment]
preferences shortcut info changed to `cmd+,`

> The only one not working so far is E (View / Show Labels) which is flakey. Either very slow, many seconds, or not responsive. Not a priority to fix I guess.

well, that's another problem. Is it the same with ps 2.3.1?

**Comment by Sirage48** [comment]
E does work fine in ps 2.3.1

**Comment by Sirage48** [comment]
Just experimented with the nightly 323 and the latest rc.
E might work initially after a second or so, and then if repeated it may work after 5 or 10 seconds.
But if I press E followed by clicking the mouse anywhere it works instantly both for show or hide labels.
Just clarifying - not a priority.

**Comment by Sirage48** [comment]
E fixed in #325
Excellent.
